### ISSUE TITLE
Add TOLNET / TOPAZ data to MELODIES MONET

### ISSUE THREAD

**Issue by rschwant** [NOAA-CSL/MELODIES-MONET#216]
TOPAZ is the same as TOLNET and the latest TOLNET data is all standardized, so this would be a good time to get this data into MELODIES MONET. TOPAZ team recommends to download their dataset from the TOLNET site.

**Comment by zmoon** [comment]
I think he had said they might change the format though, so we should check that it still works for current data.

**Comment by zmoon** [comment]
Yes when we implemented the reader the TOLNET files were a GEOMS-compliant format. I just meant I seem to recall him saying that it might change in the future.

**Comment by zmoon** [comment]
@bbakernoaa also mentioned:

MPLNET (NASA Goddard, mostly colocated with AERONET sites)
* https://mplnet.gsfc.nasa.gov/data?v=V3
* https://mplnet.gsfc.nasa.gov/mplnet_web_services.cgi?download

EARLINET (European network)
* https://www.earlinet.org/
* https://www.earlinet.org/index.php?id=125 (seems like have to use a portal to access the data)

**Comment by bbakernoaa** [comment]
Yes these are part of the GALION network.  We could think about adding all of them https://galion.world

This includes 

- AD-net https://www-lidar.nies.go.jp/AD-Net/
- EARLINET that you listed above 
- LALINET http://lalinet.org
- NDACC https://ndacc.larc.nasa.gov/data/observational-capabilities
- MPLNET
